### PR TITLE
[AMBARI-24880] [Log Search UI] Create new layout for the Audit log list.

### PR DESCRIPTION
**Pull request by tobias-istvan**
# What changes were proposed in this pull request?

A bigger change again, because the whole layout changed. There are some performance improvements. All of the solution is stepping forward to use the NGRX approach more wide.

## How was this patch tested?

It was tested manually and by unit tests:
```
Chrome 70.0.3538 (Mac OS X 10.13.6): Executed 292 of 292 SUCCESS (6.544 secs / 6.43 secs)
✨  Done in 39.26s.
```
```
PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 292 of 292 SUCCESS (11.683 secs / 11.542 secs)
✨  Done in 40.00s.
```

Please review [Ambari Contributing Guide](https://cwiki.apache.org/confluence/display/AMBARI/How+to+Contribute) before opening a pull request.

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-LogSearch-Github-PR-Builder/87/
Test FAILed.
Test FAILured.

**Comment by tobias-istvan** [comment]
retest this please

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-LogSearch-Github-PR-Builder/88/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-LogSearch-Github-PR-Builder/89/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-LogSearch-Github-PR-Builder/90/
Test PASSed.

File: ambari-logsearch-web/src/app/components/audit-logs-entries/audit-logs-entries.component.ts [hunk 1]
```diff
@@ -43,15 +46,16 @@ export class AuditLogsEntriesComponent {
   filtersForm: FormGroup;
 
   @Input()
-  totalCount: number = 0;
+  totalCount = 0;
+
+  commonFieldNames = commonFieldNames;
```
Comment (aBabiichuk) [inline ambari-logsearch-web/src/app/components/audit-logs-entries/audit-logs-entries.component.ts]: should be declared as input property

Comment (tobias-istvan) [reply ambari-logsearch-web/src/app/components/audit-logs-entries/audit-logs-entries.component.ts]: I also fixed a missing `@Input` issue.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-LogSearch-Github-PR-Builder/91/
Test PASSed.

**Pull request merged**